packages/daimo-common/src/currencies.ts

```diff
@@ -16,7 +16,8 @@ export const currencyRateUSD: CurrencyExchangeRate = {
 
 const data: [string, string, string, number][] = [
   ["Euro", "€", "EUR", 2],
-  ["Argentine Peso", "A$", "ARS", 0],
+  ["Argentine Peso", "ARS", "ARS", 0],
+  ["Ukrainian Hryvnia", "₴", "UAH", 0],
```

Comment:
these technically have 2 decimal points, but 1 UAH is worth about 2 cents so I don't think the decimals are useful